src/dynamics/video_player/player/player.js

```diff
@@ -2467,6 +2467,12 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         destroy = destroy || false;
                         this.trigger("floatingplayerclosed");
                         const floating = this.get("sticky") || this.get("floating");
+
+                        // Unset immediateOutstreamRequests on close to prevent ad retries.
+                        if (this.get("outstream")) {
+                            this.set("immediateOutstreamRequests", undefined);
```

Comment:
I would like to call `destroy` here but doing so breaks code in the [video-kargo-player](https://github.com/KargoGlobal/video-kargo-player/tree/develop) repo... Specifically these areas:

* https://github.com/KargoGlobal/video-kargo-player/blob/cfc64f3e7397a18af67b6e74b6556ae1a4972397/src/player/advanced-events.js#L774
* https://github.com/KargoGlobal/video-kargo-player/blob/cfc64f3e7397a18af67b6e74b6556ae1a4972397/src/player/ads/producer.js#L238-L245 (autoPrefetch not being defined).